decidim-core/app/controllers/decidim/participatory_process_steps_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require_dependency "decidim/application_controller"
+
+module Decidim
+  # A controller that holds the logic to show ParticipatoryProcessSteps in a
+  # public layout.
+  class ParticipatoryProcessStepsController < ApplicationController
```

Comment:
This is a copy-paste from `ParticipatoryProcessesController`, I'd move the common parts to `ApplicationController`